package.json

```diff
@@ -75,6 +75,7 @@
     "@rollup/plugin-node-resolve": "^10.0.0",
     "@rollup/plugin-typescript": "^8.2.0",
     "@tsconfig/svelte": "^1.0.10",
+    "@typechain/ethers-v5": "^6.0.0",
```

Comment:
This appears to be unused :thinking:

Reply:
weird :thinking: I first couldn't access the filters / events API of the contract and Igor pointed out that this dependency was necessary, and then I could get it working. But now I dropped it and it works without it. I must have confused something along the way. Dropped in https://github.com/radicle-dev/radicle-upstream/pull/1726/commits/fe2fd54091086cae7f2a1d0fe4ff519f12b01e2c.